elastic/kibana
# [Console Monaco] Improve highlighting for status codes in multiple responses

Closes https://github.com/elastic/kibana/issues/184010

## Summary

This PR adds color badges for the status codes in the output panel when there are multiple responses.

https://github.com/user-attachments/assets/be07b34d-7d04-4448-bf5d-b0d6fa36f2d7

**How to test:**
Send the following requests at once (try arranging them in different orders):

Success badge 🟢 : `GET _search`

Client error (warning) badge 🟡 : `GET _test`

Server error (danger) badge 🔴 : 
```
PUT /library/_bulk?refresh
{"index":{"_id":"Leviathan Wakes"}}
{"name":"Leviathan Wakes","author":"James S.A. Corey","release_date":"2011-06-02","page_count":561}
```
^ This request should usually succeed and it was fixed in https://github.com/elastic/kibana/pull/188552, but these changes aren't in this branch yet so the request still fails. I couldn't find another example that returns a 5** status code.


Note: AFAIK Es currently only uses 2**, 4**, and 5** status codes (200 OK, 201 Created, 202 Accepted, 204 No Content, 400 Bad Request, 401 Unauthorized, 403 Forbidden, 404 Not Found, 409 Conflict, 429 Too Many Requests, 500 Internal Server Error, 503 Service Unavailable). Practically only the success, warning, and danger badges will be used, but added badges for the rest status codes in case they are added in Es.
